tests/config/test_data.config

```diff
@@ -126,6 +126,8 @@ params {
                 kraken_report                                  = "${params.test_data_base}/data/genomics/sarscov2/metagenome/test_1.kraken2.report.txt"
                 krona_taxonomy                                 = "${params.test_data_base}/data/genomics/sarscov2/metagenome/krona_taxonomy.tab"
                 seqid2taxid_map                                = "${params.test_data_base}/data/genomics/sarscov2/metagenome/seqid2taxid.map"
+                nodes                                          = "${params.test_data_base}/data/genomics/sarscov2/metagenome/nodes.dmp"
+                names                                          = "${params.test_data_base}/data/genomics/sarscov2/metagenome/names.dmp"
```

Comment:
```suggestion
                nodes_dmp                                          = "${params.test_data_base}/data/genomics/sarscov2/metagenome/nodes.dmp"
                names_dmp                                          = "${params.test_data_base}/data/genomics/sarscov2/metagenome/names.dmp"
```

Reply:
I missed the test-datasets PR - have they been documended on the main README?

Reply:
I will document them now and update the PR in the test-datasets.